content/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.md

```diff
@@ -0,0 +1,71 @@
+  image: "edarras"
+image: /blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator.jpg
+---
+
+We are thrilled to announce that Kestra has raised **$3 million of funding** to make orchestration more accessible, collaborative, and scalable. 
+
+We aim to empower a wide range of users, from software engineers to data professionals, by simplifying and unifying their automation and orchestration needs. We envision Kestra as the go-to solution for organizations seeking **simple** and **versatile** orchestration.
+
+Through a seamless UI and IDE access, all workflows — from those orchestrating mission-critical applications, business processes, and data pipelines to simple automations — can be version controlled and governed using engineering best practices.
+
+## Trusted by Companies of All Sizes
+
+Since its inception, Kestra has experienced significant traction as an open-source project, thriving with a vibrant community of users and contributors. Thousands of companies, from startups to large enterprises, including **Huawei**, **Leroy Merlin**, **Acxiom**, **Tencent**, **Gorgias**, **Sophia Genetics**, and **Decathlon**, are already orchestrating their **mission-critical workloads** with Kestra.
+
+![clients](/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator/clients.png)
+
+## Breaking Down the Orchestration Silos
+
+[Kestra](https://github.com/kestra-io/kestra)’s innovative approach to orchestration aims to overcome the challenges posed by hyperspecialization in various domains — be it **data orchestration**, **process** and **microservice** orchestration, or **infrastructure** and **business automation**. Such hyperspecialization has often resulted in organizational silos, hindering collaboration and operational excellence.
+
+To address this, Kestra offers a versatile and language-agnostic developer platform while simultaneously providing an intuitive user interface tailored to business professionals. This holistic solution aims to bridge the gap between different organizational units, thereby fostering a collaborative environment conducive to greater operational efficiency.
+
+## The Orchestration Platform for Mission-Critical Operations
+
+![Kestra Baseline](/blogs/2023-10-05-announcing-Kestra-funding-to-build-the-universal-open-source-orchestrator/baseline.png)
+
+Our initial mission was to build the most efficient open-source workflow platform targeting the data engineering space. Since then, we’ve experienced tremendous growth well beyond that domain. Our users leverage Kestra not only to orchestrate data pipelines but also to power their **mission-critical applications**. They use our platform to orchestrate microservices and applications feeding data to their end products, and then they use Kestra for analytical workflows to understand how their products are performing. This way, Kestra serves as a **universal orchestration** solution for their business.
```

Comment:
```suggestion
Our initial mission was to build the most efficient open-source workflow platform targeting the data engineering space. Since then, we’ve experienced tremendous growth well beyond that domain. Our users leverage Kestra not only to orchestrate data pipelines but also to power their **business-critical applications**. They use our platform to orchestrate microservices and applications feeding data to their end products, and then they use Kestra for analytical workflows to understand how their products are performing. This way, Kestra serves as a **universal orchestration** solution for their business.
```